src/main/java/tasks/Deadlines.java

```diff
@@ -0,0 +1,29 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the deadlines that the user wants to remember.
+ */
+public class Deadlines extends Task {
```

Comment:
Would this class be better named `Deadline` instead of `Deadlines` (singular instead of plural) since it only represents a single task? Likewise for `Events` and `ToDos`